examples/src/main/java/com/opengamma/sdk/example/v3/MarginClientExample.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright (C) 2017 - present by OpenGamma Inc. and the OpenGamma group of companies
+ *
+ * Please see distribution for license.
+ */
+package com.opengamma.sdk.example.v3;
+
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.time.LocalDate;
+import java.util.Collections;
+import java.util.List;
+
+import org.joda.beans.ser.JodaBeanSer;
+
+import com.opengamma.sdk.common.auth.v3.Credentials;
+import com.opengamma.sdk.common.v3.ServiceInvoker;
+import com.opengamma.sdk.margin.v3.MarginWhatIfCalcResult;
+import com.opengamma.sdk.margin.v3.Ccp;
+import com.opengamma.sdk.margin.v3.CcpInfo;
+import com.opengamma.sdk.margin.v3.CcpsResult;
+import com.opengamma.sdk.margin.v3.MarginCalcRequest;
+import com.opengamma.sdk.margin.v3.MarginCalcResult;
+import com.opengamma.sdk.margin.v3.MarginClient;
+import com.opengamma.sdk.margin.v3.PortfolioDataFile;
+
+/**
+ * Example code for the OpenGamma Margin Service API (v3)
+ */
+public class MarginClientExample {
+
+  // credentials must be set to use the example
+  private static final Credentials CREDENTIALS = Credentials.ofApiKey("ABC", "123");
+
+  // the file to upload
+  private static final Path LCH_FILE = Paths.get("src/main/resources/lch-trades.txt");
```

Comment:
In that case it's more reliable to use the resource mechanism to load the file. Something like:

    getClass().getResource("lch-trades.txt")